get-started.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Getting Started
+summary: Quick list of instructions and hints to get started with NServiceBus and the Particular Service Platform
+reviewed: 2016-12-05
+---
+
+There is a variety of starting points for NServiceBus and the platform. The path taken will depend on an individual's specific experience and preferences for learning.
+
+
+## High Level Content
+
+These articles are recommended to gain an understanding of the key concepts and technologies used within the Platform.
+
+ * [Service Platform Overview](/platform/): Provides an overview of the various parts of Platform; [NServiceBus](/nservicebus/), [ServiceControl](/servicecontrol/), [ServiceInsight](/serviceinsight/) and [ServicePulse](/servicepulse/).
+ * [Architectural Principles](/nservicebus/architecture/principles.md)
```

Comment:
> [Architectural Principles](/nservicebus/architecture/principles.md)
  
            
  [Architectural Principles](/nservicebus/architecture/principles.md) and [Bus vs Broker architecture](/nservicebus/architecture/): A high-level overview of the messaging and the two popular architectural styles.